src/java.base/aix/native/libnio/ch/Pollset.c

```diff
@@ -26,6 +26,7 @@
 
 #include "jni.h"
 #include "jni_util.h"
+#include "jni_util_md.h"
```

Comment:
When I suggested jni_util_md.h then I meant create one for Unix and another for Windows, then have jni_util.h include it. This will avoid needing to add an include to all these files.

Reply:
I considered this too,  but the one on Windows would be empty for now -> looks a bit strange . But I can do it why not .